unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneBoundariesController/SceneBoundsChecker.cs

```diff
@@ -279,12 +279,12 @@ protected void UpdateEntityCollidersValidState(MeshesInfo meshesInfo, bool isIns
                 return;
 
             int collidersCount = meshesInfo.colliders.Count;
-            if(collidersCount <= 0)
-                 return;
-            
+            if (collidersCount <= 0)
```

Comment:
As a `Count` (or `Length`) property never can be less than 0, I think the use of `== 0` makes the code more understandable than `<= 0`.

```suggestion
            if (collidersCount == 0)
```